src/common/components/meta/PageMeta.tsx

```diff
@@ -9,22 +12,40 @@ export type Props = {
   description?: string | null;
   image?: string | null;
   canonicalUrl?: string;
-  openGraphDescription?: string;
-  openGraphTitle?: string;
-  openGraphType?: string;
-  twitterDescription?: string;
-  twitterTitle?: string;
+  openGraphDescription?: string | null;
+  openGraphTitle?: string | null;
+  openGraphType?: string | null;
+  twitterDescription?: string | null;
+  twitterTitle?: string | null;
+  localePaths?: {
+    path: string;
+    locale: Language;
+  }[];
 };
 
+export function getLanguageAwarePath(
+  locale: string | undefined,
+  path: string
+): string {
+  if (locale === 'fi') {
```

Comment:
Should this be taken from Language enum, so the places where the languages are used and checked could be found more easily?